### ISSUE TITLE
Make Formation Planes behave like Export Buses

### ISSUE THREAD

**Issue by shartte** [AppliedEnergistics/Applied-Energistics-2#6788]
Such that they are consistent with Export Buses and don't act as a Storage bus.

**Comment by Sea-Kerman** [comment]
Their current behavior needs to be maintained as an option otherwise the simple provider to interface + formation plane subnet setup won't work, and I can't think of an elegant alternative that will perform the same function of pattern provider -> item on ground

**Comment by Technici4n** [comment]
I disagree with adding lots of options to accomodate for niche cases. There's always tradeoffs.

**Comment by Sea-Kerman** [comment]
I guess the new way to do it would be provider into barrel into storage bus into formation plane

**Comment by Mecr0mage** [comment]
i would not say this is a niche case, as i have personally had many instances when i would of liked to be able to simply place a formation plane on the main network and have it export X block whenever possible, but instead i have to make an ad-hoc export buss -> interface -> formation plane. something off the top of my head is using this with an enchanted antihalation plane to quickly fortune any gem ore blocks you have in a system.

**Comment by SirOlrik** [comment]
I second on this, since both Formation plane and Annihalation plane are a AE2 devices, I would actually consider it as sutch, just like the Inscriber, Charger, Import bus, Exp... etc.
Aftert all, both planes should inherit from their Import/Export bus counterpart, so you have the bus functinality + plane functionality.
Also make the planes to be recrafted into full blocks, like the pattern provider and Energy acceptor, leaving out the hassle of having items flying around insde the cable side of the planes.
